unstructured_inference/yolox_functions.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
```

Comment:
Do we need the shebang and utf-8 encoding info at the top of this file?

Reply:
Not really, could be deleted, this file is not expected to be used standalone